roles/network_plugin/kube-router/templates/cni-conf.json.j2

```diff
@@ -16,6 +16,13 @@
 {% endif %}
             "type":"host-local"
          }
+      },
+      {
+         "type":"portmap",
+         "capabilities":{
+            "snat":true,
```

Comment:
I'm not sure `snat` is a capability, see: https://github.com/containernetworking/plugins/tree/master/plugins/meta/portmap#usage

```suggestion
         "snat":true,
         "capabilities":{
```

Reply:
I think, we can drop it away, since it is

> boolean, default true. If true or omitted, set up the SNAT chains

and all examples have it enabled. After reading documentation and testing, I see no reason to turn it off.

FYI my current rules:

```
-A CNI-DN-1372f2d50640468b6a09a -s 172.20.193.15/32 -p udp -m udp --dport 1235 -j CNI-HOSTPORT-SETMARK
-A CNI-DN-1372f2d50640468b6a09a -s 127.0.0.1/32 -p udp -m udp --dport 1235 -j CNI-HOSTPORT-SETMARK
-A CNI-DN-1372f2d50640468b6a09a -p udp -m udp --dport 1235 -j DNAT --to-destination 172.20.193.15:1235
```

172.20.193.0/24 - host-specific pod network

Reply:
I agree:

```suggestion
         "capabilities":{
```